apps/server/test/support/completion_case.ex

```diff
@@ -23,10 +23,14 @@ defmodule Lexical.Test.Server.CompletionCase do
         {DynamicSupervisor, name: Server.Project.Supervisor.dynamic_supervisor_name()}
       )
 
+    {:ok, _} = start_supervised(Lexical.RemoteControl.ProjectNodeSupervisor)
     {:ok, _} = start_supervised({Server.Project.Supervisor, project})
     Dispatch.register(project, [project_compiled()])
     RemoteControl.Api.schedule_compile(project, true)
     assert_receive project_compiled(), 5000
+
+    on_exit(fn -> RemoteControl.stop(project) end)
```

Comment:
same comment; this should not have to be stopped manually if monitoring was working correctly.